### PR TITLE
Add BigInt type tests

### LINKED ISSUE #37: BigInt type in ABIs throws an error at compile time

### PR DESCRIPTION
**Pull request by andreireanu**
Trying to fix #37

### CODE REVIEW

File: tester/core/tests/mock_call.rs [hunk 1]
```diff
@@ -946,5 +946,26 @@ async fn test_call_second_custom_enum_with_fields_arg_result() -> Result<(), Nov
...
+    let result = TesterContract::new(
+        TESTER_CONTRACT_ADDRESS
+    )
+        .call(executor.clone(), 600000000)
+        .return_big_int_arg(&BigInt::from(42i8))
+        .await?
+        .result
+        .unwrap();
+
+    let expected = BigInt::from(42 as u8);
```
Comment (gfusee) [inline tester/core/tests/mock_call.rs:966]: Can you keep the same instantiation of BigInt you did elsewhere via i8? To keep consistency

Comment (andreireanu) [reply tester/core/tests/mock_call.rs]: Sure, I missed that! All done

**Review by gfusee — APPROVED** [review]

**Comment by gfusee** [comment]
Merged! Thanks a lot for your very good work @andreireanu ! 🫶

**Comment by andreireanu** [comment]
Great! It was a pleasure as always!

**Pull request merged**